contracts/RolesRegistry/interfaces/ICommitTokensAndGrantRoleExtension.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: CC0-1.0
+
+pragma solidity 0.8.9;
+
+interface ICommitTokensAndGrantRoleExtension {
```

Comment:
In the EIP draft PR, is suggested for those who are implementing this extension to extend the EIP. Shouldn't be the case to do it here also? 
```suggestion
interface ICommitTokensAndGrantRoleExtension is IERC7589 {
```